protos/NUgus.proto

```diff
@@ -1183,11 +1217,24 @@ PROTO NUgus [
                       }
                       Camera {
                         name "left_camera"
-                        translation 0.069952 0.033795 0.06488
-                        rotation -0.570515 0.570516 0.590782 -2.07437
-                        fieldOfView 1.5707
+
+                        translation 0.08893612063588988 0.03529909622045881 0.0713674563254241
+                        rotation 0.5773189906399273 -0.5955679903440563 -0.5585709909438886 2.06585
+                        fieldOfView 1.047
```

Comment:
Is there a reason this is changed? We'd prefer the biggest fov we can get, so the old value is better?